components/IPAScheduler.tsx

```diff
@@ -0,0 +1,97 @@
+"use client"
+
+import React from "react"
+import { Card, CardContent, CardTitle } from "@/components/ui/card"
+import { SectionHeader } from "@/components/ui/section-header"
+import { GiDna2 } from "react-icons/gi"
+import CCVBars from "@/components/assets/CCVBars"
+import { Button } from "@/components/ui/button"
+
+const request_access_url =
+  "https://docs.google.com/forms/d/e/1FAIpQLSdJwbhtMznPjR6QrobBG1u-n3AMLnWdigN-NqFepsC7gbb2tw/viewform"
+
+export const IPAScheduler = () => {
+  return (
+    <section className="mt-24 mb-24">
+      {/* <SectionHeader title="Ingenuity Pathway Analysis (IPA)" align="center" /> */}
+      <div className="flex justify-center items-center h-full w-full mx-2 px-4 sm:px-8 lg:px-24">
+        <div className="flex flex-col lg:flex-row items-center justify-center w-full max-w-[2880px] gap-8 sm:gap-12 lg:gap-20">
+          {/* Text Content */}
+          <div className="w-full max-w-[721px] space-y-6">
+            <Card className="border-none shadow-none">
+              <CardTitle className="text-2xl p-6">
+                <CCVBars />
+                <h3 className="flex items-center text-[32px] pt-3">
+                  <GiDna2 className="mr-3" /> Ingenuity Pathway Analysis
+                  Scheduler
+                </h3>
+              </CardTitle>
+              <CardContent className="pt-4 space-y-4 text-xl">
+                <p>
+                  IPA helps you identify biological mechanisms and pathways
+                  relevant to your research data. Visualize, analyze, and
+                  understand molecular interactions and disease on multiple
+                  levels.
+                </p>
+                <div className="flex justify-center items-center p-2">
+                  <Button
+                    className="h-[55px] font-semibold"
+                    variant="primary_filled"
+                  >
+                    <a
+                      href={request_access_url}
+                      target="_blank"
+                      rel="noopener noreferrer"
+                    >
+                      Request Access to IPA
+                    </a>
+                  </Button>
+                </div>
+                <ul className="list-disc pl-6 space-y-2">
+                  <h4 className="text-2xl font-semibold italic">Scheduling</h4>
```

Comment:
Was this text from the CBC website? Or wherever the IPA scheduler use to be? When we chat with Ashok, we can review the language